fingerprint/capture/build.gradle.kts

```diff
@@ -38,6 +38,7 @@ dependencies {
 
     // Android X
     implementation(libs.androidX.ui.viewpager2)
+    implementation(libs.androidX.ui.preference)
```

Comment:
Why do you need preferences UI in this module?

Reply:
To access the PrefsManger class as it is part of the androidx prefs lib

Reply:
Would it make sense to decouple the actual setting from the UI?